src/java/org/apache/cassandra/io/util/FileHandle.java

```diff
@@ -442,6 +449,9 @@ else if (mmapped)
                 }
                 else
                 {
+                    if (adviseRandom)
+                        logger.warn("adviseRandom ignored for non-mmapped FileHandle {}", file);
```

Comment:
is this going to be logged for every FileHandle not using mmap ? like when you use NSS

Reply:
My goal is to make it obvious when you're asking for `adviseRandom` and it's not supported, instead of silently doing nothing.  How would you suggest achieving this while playing nice with NSS?